drill-admin/src/main/kotlin/com/epam/drill/endpoints/plugin/PluginDispatcher.kt

```diff
@@ -4,6 +4,8 @@ package com.epam.drill.endpoints.plugin
 import com.epam.drill.common.AgentInfo
 import com.epam.drill.common.Message
 import com.epam.drill.common.MessageType
+import com.epam.drill.common.MessageWithId
+
```

Comment:
why?

Reply:
do you mean the empty raw or the MessageWithId import